prisma/seed.js

```diff
@@ -49,25 +49,24 @@ async function seed() {
       cohorts.push(cohort)
     }
 
-    try {
-      const user = await prisma.user.create({
-        data: {
-          email: `test${i}@test.com`,
-          password,
-          cohortId: cohorts[0].id,
-          profile: {
-            create: {
-              firstName: `name${i}`,
-              lastName: `surname${i}`,
-              bio: `Here i am, coding like a hurricane`,
-              profileImageUrl: profileImages[i]
-            }
+    const user = await prisma.user.create({
+      data: {
+        email: `test${i}@test.com`,
+        password,
+        cohortId: cohorts[0].id,
+        ...(i === 3 && { isActive: false }),
+        profile: {
+          create: {
+            firstName: `name${i}`,
+            lastName: `surname${i}`,
+            bio: `Here i am, coding like a hurricane`,
+            profileImageUrl: profileImages[i]
           }
         }
-      })
-      users.push(user)
-      myEmitter.emit('register', users[0])
-    } catch (err) {}
+      }
+    })
+    myEmitter.emit('register', user)
+    users.push(user)
   }
 
   const createdUser = await prisma.user.create({
```

Comment:
It might be better if you split creating a user out into a function, then you won't have to use magic numbers like `users[3]`. You could just pass the user data through the the function.